.eslintrc.js

```diff
@@ -1,21 +1,84 @@
 module.exports = {
-  extends: "eslint:recommended",
+ 'extends': [
+   'eslint-config-airbnb-base',
+   'eslint-config-airbnb-base/rules/strict',
+ ],
   parserOptions: {
     ecmaVersion: 6,
-    sourceType: "module",
+    sourceType: 'module',
     ecmaFeatures: {
       impliedStrict: true
     }
   },
+ settings: {
+     'import/resolver': {
+         'webpack': {}
+     }
+ },
   env: {
     browser: true,
     es6: true,
     amd: true,
     commonjs: true
   },
   rules: {
-    // Stylistic rules
-    "eol-last": "error",
-    "no-trailing-spaces": "error",
+    // TODO reactivate all these rules
+
+    // The following rule should be no more restrictive than
+    //'no-plusplus': ['error', { 'allowForLoopAfterthoughts': true }],
+    // to allow for i++ in for loops (even though we should mostly use for..of
+    // anyway)
+    'no-plusplus': 'off',
+
+    'func-names': 'off',
+    'no-mixed-operators': 'off',
+    'no-use-before-define': 'off',
+    'no-underscore-dangle': 'off',
```

Comment:
In Itowns, we had used this one as a way of saying "this field in private". I'm in favor of keeping this though, because usually we can enforce the privacy of something by moving it in a closure or outside of the exported object.